extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -147,19 +163,29 @@ If you're a vendor that does _not_ provide a [custom video player component](../
 By hosting a generic [integration document](#integration) that can reference videos with URL parameters, authors don't need to provide the inner player document themselves, but only include an `<amp-video-iframe>` tag in the AMP document:
 
 ```html
+<!--
+  data-param-* attributes are added to src and poster, so this would use the src
+  https://vendor.example/amp-video-iframe
+    ?videoid=MY_VIDEO_ID
+    &channelid=MY_CHANNEL_ID
+-->
 <amp-video-iframe
   layout="responsive"
   width="16"
   height="9"
-  src="https://video-provider.example/amp-video-iframe.html?videoid=PROVIDED_VIDEO_ID"
-  poster="https://video-provider.example/amp-video-iframe-poster.jpg?videoid=PROVIDED_VIDEO_ID"
+  src="https://vendor.example/amp-video-iframe"
+  poster="https://vendor.example/poster.jpg"
+  data-param-videoid="MY_VIDEO_ID"
+  data-param-channelid="MY_CHANNEL_ID"
 >
 </amp-video-iframe>
 ```
 
-The dynamic server-side document `amp-video-iframe.html` includes the video as necessary per the configurable `PROVIDED_VIDEO_ID` parameter. This document bootstraps the [iframe integration script](#integration) so that the AMP document can coordinate with the player.
+The `src` and `poster` URLs are appended with [`data-param-*` attributes as query string](#data-param).
```

Comment:
What if the `post` and `src` need different names for the same parameter 🤔

Reply:
Up to user to either normalize or repeat arg :)

Reply:
SGTM and to be sure, there are no existing `data-params` (global or `amp-video-iframe` specific) that might unintentionally get added to the urls as a breaking change right?

Reply:
The assumption is that the `data-param-` prefix should suffice to prevent unintentional attributes from being added as params. If they are, the last resort is that superfluous query params in a URL are likely to be meaningless.